packages/sdk/shared/src/types/botEndpoint.ts

```diff
@@ -36,6 +36,7 @@ export interface BotEndpoint {
   botUrl: string;
   msaAppId: string;
   msaPassword: string;
+  tenantId: string;
```

Comment:
Should this be optional?
```suggestion
  tenantId?: string;
```

Reply:
shouldnt this be `msaTenantId`?

Reply:
fixed and tested 
![image](https://github.com/southworks/BotFramework-Emulator/assets/122501764/4d950280-6e99-4d35-8902-bd40f15a9703)